server/planning/planning_notifications.py

```diff
@@ -304,7 +304,7 @@ def _send_user_email(user_id, contact_id, source, meta_message, data):
 
     admins = app.config["ADMINS"]
 
-    data["subject"] = "Superdesk assignment" + ": {}".format(data.get("slugline") if data.get("slugline") else "")
+    data["subject"] = render_template("assignment_mail_subject" + ".txt", **data)
```

Comment:
not sure why this string concatenation, you can just do `"assignment_mail_subject.txt"`